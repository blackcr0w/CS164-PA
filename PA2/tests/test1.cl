/home/ff/cs164/etc/../assignments/PA2/tests/test1.cl