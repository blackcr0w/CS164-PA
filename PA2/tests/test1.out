#name "tests/test1.cl"
#1 '('
#1 '*'
#1 OF
#1 '*'
#1 ')'
#1 INT_CONST 0
#1 '('
#1 '*'
#1 '*'
#1 ')'
#1 INT_CONST 1
#1 '('
#1 '*'
#1 '*'
#1 ')'
#1 INT_CONST 2
#1 '('
#1 '*'
#1 '*'
#1 ')'
#1 INT_CONST 3
#1 '('
#1 '*'
#1 '*'
#1 ')'
#1 INT_CONST 4
#1 '('
#1 '*'
#1 '*'
#1 ')'
#1 INT_CONST 5
LEXER BUG - UNMATCHED: v
LEXER BUG - UNMATCHED: a
LEXER BUG - UNMATCHED: r
LEXER BUG - UNMATCHED: i
LEXER BUG - UNMATCHED: o
LEXER BUG - UNMATCHED: u
LEXER BUG - UNMATCHED: s
LEXER BUG - UNMATCHED: f
LEXER BUG - UNMATCHED: o
LEXER BUG - UNMATCHED: r
LEXER BUG - UNMATCHED: m
LEXER BUG - UNMATCHED: s
LEXER BUG - UNMATCHED: w
LEXER BUG - UNMATCHED: h
LEXER BUG - UNMATCHED: i
LEXER BUG - UNMATCHED: t
LEXER BUG - UNMATCHED: e
LEXER BUG - UNMATCHED: s
LEXER BUG - UNMATCHED: p
LEXER BUG - UNMATCHED: a
LEXER BUG - UNMATCHED: c
LEXER BUG - UNMATCHED: e
LEXER BUG - UNMATCHED: b
LEXER BUG - UNMATCHED: l
LEXER BUG - UNMATCHED: a
LEXER BUG - UNMATCHED: n
LEXER BUG - UNMATCHED: k
LEXER BUG - UNMATCHED: s
LEXER BUG - UNMATCHED: f
LEXER BUG - UNMATCHED: o
LEXER BUG - UNMATCHED: r
LEXER BUG - UNMATCHED: m
LEXER BUG - UNMATCHED: f
LEXER BUG - UNMATCHED: e
LEXER BUG - UNMATCHED: e
LEXER BUG - UNMATCHED: d
LEXER BUG - UNMATCHED: c
LEXER BUG - UNMATCHED: a
LEXER BUG - UNMATCHED: r
LEXER BUG - UNMATCHED: r
LEXER BUG - UNMATCHED: i
LEXER BUG - UNMATCHED: a
LEXER BUG - UNMATCHED: g
LEXER BUG - UNMATCHED: e
LEXER BUG - UNMATCHED: r
LEXER BUG - UNMATCHED: e
LEXER BUG - UNMATCHED: t
LEXER BUG - UNMATCHED: u
LEXER BUG - UNMATCHED: r
LEXER BUG - UNMATCHED: n
LEXER BUG - UNMATCHED: t
LEXER BUG - UNMATCHED: a
LEXER BUG - UNMATCHED: b
LEXER BUG - UNMATCHED: v
LEXER BUG - UNMATCHED: e
LEXER BUG - UNMATCHED: r
LEXER BUG - UNMATCHED: t
LEXER BUG - UNMATCHED: i
LEXER BUG - UNMATCHED: c
LEXER BUG - UNMATCHED: a
LEXER BUG - UNMATCHED: l
LEXER BUG - UNMATCHED: t
LEXER BUG - UNMATCHED: a
LEXER BUG - UNMATCHED: b
